presto-main/src/main/java/com/facebook/presto/server/QueryExecutionResource.java

```diff
@@ -102,7 +102,7 @@ public Response getTaskInfo(@PathParam("queryId") String queryId)
                     }
                 }
 
-                long last = System.currentTimeMillis();
+                long last = TimeUnit.NANOSECONDS.toMillis(System.nanoTime());
```

Comment:
I changed this because even though it's not directly used in a `Duration` object it is used to measure a duration on line 113. I traced the second term in that subtraction to originate from `System.nanoTime()` and thought that the first term should also derive from `System.nanoTime()`.